contents/docs/web-analytics/web-vitals.mdx

```diff
@@ -0,0 +1,132 @@
+
+A good LCP score is 2.5 seconds or less. If your LCP is higher, users might perceive your site as slow or unresponsive. Common causes of poor LCP include slow server response times, render-blocking JavaScript and CSS, slow resource load times, and client-side rendering.
+
+
+### FCP - First Contentful Paint
+
+First Contentful Paint (FCP) measures the time from when the page starts loading to when any part of the page's content is rendered on the screen. This could be text, images, SVGs, or even `<canvas>` elements. FCP is important because it indicates when users first see something happening on your page, giving them confidence that the page is actually loading.
+
+A good FCP score is 1.8 seconds or less. Poor FCP scores can be caused by render-blocking resources (like CSS or JavaScript files), slow server response times, or large HTML documents that take time to parse. Improving FCP often involves optimizing critical rendering paths, reducing server response times, and minimizing or deferring non-critical resources.
+
+
+### CLS - Cumulative Layout Shift
+
+Cumulative Layout Shift (CLS) measures visual stability by quantifying how much unexpected layout shifts occur during the entire lifespan of the page. A layout shift happens when a visible element changes its position from one rendered frame to the next. CLS is important because unexpected movement of page content can be extremely frustrating for users, especially if they were about to click or tap on something that suddenly moves.
+
+A good CLS score is 0.1 or less. Common causes of poor CLS include images or ads without dimensions, dynamically injected content, web fonts causing FOIT/FOUT (Flash of Invisible/Unstyled Text), and actions waiting for a network response before updating the DOM. To improve CLS, always include width and height attributes on images and video elements, reserve space for dynamic content, and ensure web fonts don't cause layout shifts. Modern web frameworks should help you with this.
+
+
+### INP - Interaction to Next Paint
+
+Interaction to Next Paint (INP) measures the responsiveness of your site by measuring the time it takes to respond to user interactions like clicks or taps. It's important because it tells you how quickly your site reacts to user input, which is a key factor in user experience.
+
+A good INP score is 100ms or less. Poor INP scores can be caused by slow JavaScript execution, blocking the main thread, or excessive layout thrashing. To improve INP, optimize JavaScript performance, reduce unnecessary layout changes, and ensure smooth animations.
+
+## Web vitals dashboard
+
+You can access web vitals in the sidebar by clicking on the dropdown to the right of web analytics.
+
+If you're inside web analytics, you can also toggle to the web vitals view by clicking the toggle in the top right.
+
+> TODO: ADD IMAGE
```

Comment:
Missing this, working on it, needs to merge https://github.com/PostHog/posthog/pull/27691 first